<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bolus</class>
 <widget class="QWidget" name="bolus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="CarbsLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LogoBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculationBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>411</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>View Calculation</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="CalculationDisplay">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="GlucoseLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>GLUCOSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="CarbsLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>CARBS</string>
   </property>
  </widget>
  <widget class="QLabel" name="UnitsLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>units</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="GlucoseLineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="ExtendedBolusRadio">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Extended Bolus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmBtn">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="NowLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Deliver Now</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NowLineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>480</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="LaterLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>450</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Deliver Later</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LaterLineEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>480</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="DurationDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>0.500000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="DurationLabel">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExtendedConfirmBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentageLabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>490</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentagleLabel2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="hrsLabel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>hrs</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
